core/domain/exp_domain.py

```diff
@@ -1298,6 +1318,29 @@ def is_demo(self):
         """
         return self.is_demo_exploration_id(self.id)
 
+    @property
+    def is_stale(self):
+        """Whether the exploration proto size is up to date.
+
+        Returns:
+            int. Updated exploration's proto size.
+        """
+        if self.proto_size_is_stale:
+            self._cached_android_proto_size_in_bytes = self.get_proto_size()
+            self.proto_size_is_stale = False
+
+        return self._cached_android_proto_size_in_bytes
+
+    def __setattr__(self, aa, new_value):
```

Comment:
Please don't name it 'aa'. That was just example code. Pick a name that has a clear meaning and document this method carefully as well.

Reply:
Yeah, that was missed in hurry. Fixed now.